### DISCUSSION TITLE
Should alliances be restricted to only one victory condition?

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [ihhub/fheroes2#9104] [General]
In the editor alliances are only available when the win condition i set to _One alliance defeats another_. However there could be many intresting options where the player has another win condition but sill fights against a team of players. Now that the alliance feature is already in the game, in my mind it doesn't seams that it would require that much work to addtt the alliance feature to also work with other win conditions.

**Comment by ihhub** [comment]
It is easy to say but it is not easy to implement. We have plans to expand the functionality of alliances but it is not trivial as it seems.

**Reply by Mr-Bajs** [reply]
Okey. Great. So it's on the roadmap.
